### ISSUE TITLE
TypeError: Invalid Version: undefined with Ember 2.11 beta

### ISSUE THREAD

**Issue by ochervak** [offirgolan/ember-light-table#295]
```
TypeError: Invalid Version: undefined
    at new SemVer (/Users/ochervak/Projects/ember-quickstart/node_modules/semver/semver.js:281:11)
    at compare (/Users/ochervak/Projects/ember-quickstart/node_modules/semver/semver.js:571:10)
    at Function.lt (/Users/ochervak/Projects/ember-quickstart/node_modules/semver/semver.js:605:10)
    at BowerDependencyVersionChecker.DependencyVersionChecker.(anonymous function) [as lt] (/Users/ochervak/Projects/ember-quickstart/node_modules/ember-scrollable/node_modules/ember-cli-version-checker/index.js:68:26)
    at Class.treeFor (/Users/ochervak/Projects/ember-quickstart/node_modules/ember-scrollable/index.js:27:31)
    at Class.superWrapper [as treeFor] (/Users/ochervak/Projects/ember-quickstart/node_modules/core-object/lib/assign-properties.js:32:18)
    at /Users/ochervak/Projects/ember-quickstart/node_modules/ember-cli/lib/models/addon.js:245:34
    at Array.map (native)
    at Class.eachAddonInvoke (/Users/ochervak/Projects/ember-quickstart/node_modules/ember-cli/lib/models/addon.js:243:24)
```

**Comment by offirgolan** [comment]
Looks like you are having some dependency issues there that arent related to this addon. Please run the following commands to clean your directory and re-install your dependencies.

```
npm cache clear && bower cache clean && rm -rf node_modules bower_components dist tmp
npm i && bower i
```

**Comment by efx** [comment]
This was an issue with ember-scrollables dependency that has since been fixed. See https://github.com/alphasights/ember-scrollable/issues/36.

**Comment by HenryVonfire** [comment]
@efx @offirgolan still happens to me. Isn't `ember-light-table` using the latest version of `ember-scrollable`?
```
Invalid Version: undefined
TypeError: Invalid Version: undefined
    at new SemVer ([project]/node_modules/ember-scrollable/node_modules/semver/semver.js:281:11)
    at compare ([project]/node_modules/ember-scrollable/node_modules/semver/semver.js:571:10)
    at Function.lt ([project]/node_modules/ember-scrollable/node_modules/semver/semver.js:605:10)
    at DependencyVersionChecker.(anonymous function) [as lt] ([project]/node_modules/ember-scrollable/node_modules/ember-cli-version-checker/index.js:68:26)
    at Class.treeFor ([project]/node_modules/ember-scrollable/index.js:27:31)
    at Class.superWrapper [as treeFor] ([project]/node_modules/core-object/lib/assign-properties.js:32:18)
    at [project]/node_modules/ember-cli/lib/models/addon.js:245:34
    at Array.map (native)
    at Class.eachAddonInvoke ([project]/node_modules/ember-cli/lib/models/addon.js:243:24)
    at Class.treeFor ([project]/node_modules/ember-cli/lib/models/addon.js:329:22)

```

**Comment by efx** [comment]
@HenryVonfire in some combinations it should I think: https://github.com/offirgolan/ember-light-table/blob/master/package.json#L71 
as a workaround you can just update light-table's package.json in your project

**Comment by hoboman313** [comment]
Ember 2.11.0 is out this morning and I'm running to the same issue with ember-light-table/ember-scrollable now

**Comment by jamesdixon** [comment]
Same situation as @hoboman313

**Comment by offirgolan** [comment]
Working on it! Stay tuned 😄

**Comment by luketheobscure** [comment]
If anyone is getting this and using `npm shrinkwrap`, you can fix it by manually bumping the ember-light-table > ember-scrollable >ember-cli-version-checker to 1.2.0 in you `npm-shrinkwrap.json` file.

**Comment by offirgolan** [comment]
There are a couple of PRs open on ember-scrollable that should hopefully merged asap. Once those are in, a new release will be made and the issue should be resolved. Sorry for the hiccup on this guys.

**Comment by offirgolan** [comment]
Fixed and released with `v1.8.2`.
